woocommerce/woocommerce
# Fix deleting a tax class via WP CLI.

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds `GET` support for an individual tax class and fixes deleting a tax class via WP CLI 
Closes #27215 .

### How to test the changes in this Pull Request:

1. Create two new test tax classes via the REST API (eg. `{"name":"Sample Tax"}`)
2. Retrieve the tax classes to verify `/wp-json/wc/v3/taxes/classes`
3. `wp wc tax_class list --user=1`
4. `wp wc tax_class delete example-tax --force=true --user=1`
5. Check both REST API and WP CLI to verify the tax class was removed

### Other information:




### Changelog entry

> Fix - Fixed WP CLI command to delete tax classes.
